cli/server_create_admin_user.go

```diff
@@ -0,0 +1,262 @@
+//go:build !slim
```

Comment:
Typically in the codebase we've reserved `_` to denote build tags. I'd like to keep that pattern going (or change it entirely, but not diverge in a single change).

It's a bit weird but I think better to do `server_createadminuser`. Still doesn't follow build tags, but maybe a bit more so with the existing style.

Reply:
There's an argument to be made that our top-level admin commands should be moved under a `admin` subcommand:

- scaletest
- create-admin-user
- state

Doesn't have to be in this PR though...

Reply:
Renamed the file to `server_createadminuser.go`